ql/src/test/results/clientnegative/materialized_view_no_tbl.q.out

```diff
@@ -0,0 +1 @@
+FAILED: SemanticException Materialized view must have a table defined.
```

Comment:
The PR seems to be missing the corresponding `materialized_view_no_tbl.q` file.

Reply:
DURRRR!!!  Noob mistake.  Not sure how I missed this!  Will add on next patch.